mne_connectivity/utils/docs.py

```diff
@@ -424,7 +426,7 @@
 """
 
 docdict["axes_topomap"] = """
-axes : length-2 tuple of list of matplotlib.axes.Axes | None (default None)
+axes : tuple of length 2 of list of matplotlib.axes.Axes | None (default None)
```

Comment:
here I'd do as elsewhere and just say "tuple of length 2". The docstring already explains that "there must be two lists containing the aes for the seeds and targets"

```suggestion
axes : tuple of length 2 | None (default None)
```